superset/assets/src/dashboard/stylesheets/builder-sidepane.less

```diff
@@ -1,55 +1,62 @@
 .dashboard-builder-sidepane {
-  background: white;
-  flex: 0 0 376px;
-  border: 1px solid @gray-light;
+  flex: 0 0 374px;
   z-index: 10;
   position: relative;
+  overflow-x: hidden;
+  box-shadow: -4px 0 4px 0 rgba(0, 0, 0, 0.1);
 
   .dashboard-builder-sidepane-header {
     font-size: 15px;
     font-weight: 700;
+    border-top: 1px solid @gray-light;
     border-bottom: 1px solid @gray-light;
-    padding: 14px;
+    padding: 16px;
   }
 
   .trigger {
-    height: 25px;
+    height: 18px;
     width: 25px;
-    color: @gray;
-    position: relative;
-
-    &.close {
-      top: 3px;
-    }
-
-    &.open {
-      position: absolute;
-      right: 14px;
-    }
+    color: @almost-black;
+    float: none;
+    opacity: 1;
   }
 
-  .component-layer {
-    .new-component.static {
-      cursor: pointer;
-    }
+  .viewport {
+    position: absolute;
+    background: white;
+    overflow: hidden;
+    width: 374px;
```

Comment:
do you think it'd be better to define this as a variable in `variables.less`? then it's less error prone to changes, eg slider container could be `(2 * width)`

Reply:
yes. use variable is much better.